### DISCUSSION TITLE
Smaller subscription plan

### DISCUSSION THREAD

**Discussion by viewsourc3** [plausible/analytics#1882] [Anything Else]
Hello, 

I was wondering how you'd feel creating a smaller subscription plan. I have a website that should attract less than a 1000 visitors per week. It's a personal page with no generated profits. I am however interested in having high level metrics with regards to visits.  

The first subscription plan covers up to 10K visits/month. But dedicating +100 euros/year seems a little too much for that kind of website.

Have you considered offering a lower subscription plan for those like me? 

Thanks!

**Comment by metmarkosaric** [comment]
thanks for the feedback @viewsourc3! 

our up to 10k pageviews/month tier is meant for sites like yours. we've tried to make our plans as fair and as affordable as possible. if you choose to subscribe to the full year, we give you 2 months free. you can add multiple websites to the same account too. 

we don't sell your data so subscription fees is how we pay our bills and salaries. and running our infrastructure, adding/maintaining features, having customer support etc takes a lot of time and resources. we have other expenses to pay such as the fees that our payment processor charges which makes it difficult for us to lower the prices and stay sustainable.

perhaps as we grow we might be able to do something different in terms of tiers/prices but no promises at this point. sorry about that and thanks for understanding!

**Comment by viewsourc3** [comment]
Thanks for the quick reply! 
I support OSS as much as I can and I definitely appreciate the privacy-centered approach. I thought it was worth asking, just in case. Thank you

**Reply by metmarkosaric** [reply]
you're welcome!
